src/pages/Login/Login.js

```diff
@@ -1,7 +1,85 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
 
 function Login() {
-  return <div>Login</div>;
+  const [id, setId] = useState(' ');
+  const [pw, setPw] = useState(' ');
+  const navigate = useNavigate();
+
+  const getUserId = e => {
+    setId(e.target.value);
+  };
+
+  const getUserPw = e => {
+    setPw(e.target.value);
+  };
+
+  const goToMain = () => {
+    navigate('/main');
+  };
+
+  const goToSignUp = () => {
+    navigate('/signup');
+  };
+
+  const isDisabled = id.length > 0 && pw.length > 1;
+
+  return (
+    <div>
```

Comment:
```suggestion
    <div className="login">
```
최상위 태그는 컴포넌트 이름과 같도록 수정 해 주세요

Reply:
네 반영하였습니다!